src/adagrams.js

```diff
@@ -1,8 +1,125 @@
+const letterPool = {
+  alphabet: ['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z'],
+  quantities: [9,2,2,4,12,2,3,2,9,1,1,4,2,6,8,2,1,6,4,6,4,2,2,1,2,1],
+  build () {
+    let letterPool = [];
+
+    const fillLetterPool = () => {
+      for (const index in this.alphabet) {
+        for (let i=0; i < this.quantities[index]; i++) {
+          letterPool.push(this.alphabet[index]);
+      }}
+    return letterPool;
+    };
+
+    fillLetterPool();
+
+    return letterPool;
+  },
+}
+
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+    let letters = letterPool.build();
+    let hand = [];
+    
+    for (let i=0; i < 10; i++) {
+      hand.push(letters[Math.floor(Math.random() * letters.length)]); 
+    }
```

Comment:
This is a good start, but because you aren't modifying the pool after each draw, it's possible (however unlikely) that a player ends up with a hand made entirely of 'Q's.

Look up the `splice` function.